Parse/PFQuery.h

```diff
@@ -197,6 +197,17 @@ typedef void (^PFQueryArrayResultBlock)(NSArray<PFGenericObject> *_Nullable obje
 - (instancetype)whereKey:(NSString *)key notEqualTo:(id)object;
 
 /**
+ Add a constraint for finding string values that contain a provided
+ string using Full Text Search
+
+ @param key The key to be constrained.
+ @param value the substring that the value must contain.
+
+ @return The same instance of `PFQuery` as the receiver. This allows method chaining.
+ */
+- (instancetype)whereKey:(NSString *)key fullText:(NSString *)text;
```

Comment:
Can we call it matchesText instead of fullText?

Reply:
done